action/envelope.go

```diff
@@ -88,6 +88,20 @@ func (elp *Envelope) Proto() *iotextypes.ActionCore {
 		actCore.Action = &iotextypes.ActionCore_DepositToRewardingFund{DepositToRewardingFund: act.Proto()}
 	case *PutPollResult:
 		actCore.Action = &iotextypes.ActionCore_PutPollResult{PutPollResult: act.Proto()}
+	case *StakeCreate:
+		actCore.Action = &iotextypes.ActionCore_StakeCreate{StakeCreate: act.Proto()}
+	case *StakeUnstake:
+		actCore.Action = &iotextypes.ActionCore_StakeUnstake{StakeUnstake: act.Proto()}
+	case *StakeWithdraw:
+		actCore.Action = &iotextypes.ActionCore_StakeWithdraw{StakeWithdraw: act.Proto()}
+	case *StakeAdd:
+		actCore.Action = &iotextypes.ActionCore_StakeAddDeposit{StakeAddDeposit: act.Proto()}
+	case *StakeAgain:
+		actCore.Action = &iotextypes.ActionCore_StakeRestake{StakeRestake: act.Proto()}
+	case *StakeChangeCandidate:
+		actCore.Action = &iotextypes.ActionCore_StakeChangeCandidate{StakeChangeCandidate: act.Proto()}
+	case *StakeTransferOwnership:
+		actCore.Action = &iotextypes.ActionCore_StakeTransferOwnership{StakeTransferOwnership: act.Proto()}
```

Comment:
About naming:
Stake, Unstake, Restake, WithdrawStake, SwitchCandidate, TransferStake, DepositToStake

Reply:
yes! renamed it